src/main/java/org/spongepowered/api/text/Texts.java

```diff
@@ -175,25 +203,25 @@ public static Text of() {
      * @return The built text object
      */
     public static Text of(Object... objects) {
+        // Shortcut for lonely TextRepresentables
+        if (objects.length == 1 && objects[0] instanceof TextRepresentable) {
```

Comment:
For example, take a player that I want to display the name of. I have no idea what the TextRepresentable that it extends entails (nothing says so in javadocs), so I have to handle the formatting manually:

```java
Texts.of("Hello ", player.get(Keys.DISPLAY_NAME).or(Texts.of()))
```

In this case, TextRepresentable hasn't actually helped me achieve my goal of formatting the player, I still have to do it manually every time.